### DISCUSSION TITLE
FTP Plugin can't connect to the server with Ipv6 address

### DISCUSSION THREAD

**Discussion by aabm197** [doublecmd/doublecmd#1483] [General]
When the Host address is in Ipv6, the plugin fails to connect. Can the plugin be made Ipv6 enabled in future updates?

**Comment by alexx2000** [comment]
Done.

**Reply by aabm197** [reply]
Thank you for your kind and prompt action

**Comment by aabm197** [comment]
Will somebody be kind enough to guide me on how to update the plugin now?
I am running Double Commander Version: 1.0.11 Revision: 272 on Ubuntu Linux-gtk2

**Reply by j2969719** [reply]
https://doublecmd.github.io/doc/en/faq.html#repo
or
https://doublecmd.sourceforge.io/site/eng/snapshots.php

**Reply by aabm197** [reply]
Installed version 1.2.0 alpha.
With Host address in Ipv6, the FTP plugin successfully connects to the server.
Many many thanks to everybody for the kind support.
